flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/cli/SqlCommandParser.java

```diff
@@ -207,6 +301,10 @@ public String toString() {
 		public boolean hasOperands() {
 			return operandConverter != NO_OPERANDS;
 		}
```

Comment:
We should use equals instead of `!=` to compare the operands.

Reply:
`NO_OPERANDS ` is internal variable and NO_OPERANDS is a static final instance. we can use `!=`  to do compare